### PR TITLE
added _temporal_split_from_datetime method

### PR DESCRIPTION
**Pull request by atakanserifoglu**

### CODE REVIEW

**Comment by Konuralpkilinc** [comment]
LGTM!

File: handler.py [hunk 1]
```diff
@@ -372,6 +373,24 @@ def main():
 
         except (ValueError, PermissionError) as e:
             print(f"Error: {e}")
-
+            
+def _temporal_split_from_datetime(
+    example_datetime: datetime.datetime
+) -> Optional[str]:
```
Comment (atakanserifoglu) [inline handler.py:379]: zoer

File: handler.py [hunk 2]
```diff
@@ -372,6 +373,24 @@ def main():
 
         except (ValueError, PermissionError) as e:
             print(f"Error: {e}")
-
+            
+def _temporal_split_from_datetime(
+    example_datetime: datetime.datetime
```
Comment (atakanserifoglu) [inline handler.py:378]: deneme yorumudur

File: handler.py [hunk 3]
```diff
@@ -372,6 +373,24 @@ def main():
...
+    if example_datetime < end_train_datetime:
+        return 'train'
+    elif example_datetime < end_valid_datetime:
+        return 'validation'
+    elif example_datetime < end_test_datetime:
+        return 'test'
+    else:
+        return None
+    
+
```
Comment (atakanserifoglu) [inline handler.py:394]: deneme

**Review by atakanserifoglu — COMMENTED** [review]
!cond_accept

**Pull request closed without merging**